CHANGELOG.md

```diff
@@ -1,3 +1,8 @@
+### UNRELEASED
```

Comment:
```suggestion
## UNRELEASED
```

for the parsing to work, @boutell could you confirm that we should not use 3 `#` anymore?

Reply:
Use two

Reply:
Changed it to two